includes/core-changes/aspnetcore/5.0/kestrel-disables-http2-over-tls.md

```diff
@@ -0,0 +1,69 @@
+### Kestrel: HTTP/2 disabled over TLS on incompatible Windows versions
+
+To enable HTTP/2 over Transport Layer Security (TLS) on Windows, two requirements need to be met:
+
+- Application-Layer Protocol Negotiation (ALPN) support, which is available starting with Windows 8.1 and Windows Server 2012 R2.
+- A set of ciphers compatible with HTTP/2, which is available starting with Windows 10 and Windows Server 2016.
+
+As such, Kestrel's behavior when HTTP/2 over TLS is configured has changed to:
+
+- Downgrade to `Http1` and log a message at the `Information` level when [ListenOptions.HttpProtocols](https://github.com/dotnet/aspnetcore/blob/c4c6e25a0660ffbcae2d81c979495ec9533ccaff/src/Servers/Kestrel/Core/src/ListenOptions.cs#L79) is set to `Http1AndHttp2`. `Http1AndHttp2` is the default value for `ListenOptions.HttpProtocols`.
```

Comment:
Just curious, why is this a source link rather than API ref?

Reply:
Thank you for catching that. I simply forgot to replace it.